src/demo-app/select/select-demo.html

```diff
@@ -129,5 +129,32 @@
     </md-card>
   </div>
 
+  <div *ngIf="showSelect">
+    <md-card>
+      <md-card-subtitle>compareWith</md-card-subtitle>
+      <md-card-content>
+        <md-select placeholder="Drink" [color]="drinksTheme"
+                   [ngModel]="currentDrinkObject"
+                   (ngModelChange)="setDrinkObjectByCopy($event)"
```

Comment:
Hmm, this is a bit odd to use for manual testing because when comparing by reference, it looks broken. I think it might be clearer if you use a regular two-way binding on `ngModel`, then have a button that says "REASSIGN DRINKS" or "COPY DRINKS" to test whether the value clears.